browser-test/src/support/admin_programs.ts

```diff
@@ -651,6 +651,28 @@ export class AdminPrograms {
     }
   }
 
+  /**
+   * Add questions to specified block. You must already be on the admin program edit block page,
+   * but if you are this is a significantly faster way to add multiple questions since it does
+   * not return to the program list page each time and navigate back to the block.
+   * @param {BlockSpec} block Block information
+   */
+  async addQuestionsToProgramBlock(block: BlockSpec) {
```

Comment:
Should this method name be suffixed with "alreadyOnEditBlockPage" like addProgramBlockUsingSpecAlreadyOnEditDraftPage below? The jsdoc does say you must already be on the admin program edit block page, but it reads more as a "by the way" than a hard prerequisite.

Reply:
I'm only just starting to push jsdoc so most things lack them. I don't know if people will be completely aware to check at this time. But I'll make the method be 'WhenAlreadyOnDraftPage'.